.github/workflows/devbuild.yml

```diff
@@ -107,17 +107,31 @@ jobs:
         sudo apt-get -qy install \
             sudo curl git build-essential make cmake libc6-dev gcc g++ \
             python3 python3-dev python3-venv
-        # Install qt6 only with ubuntu-22.04
-        # This page explains why we need libgl1-mesa-dev
-        # https://doc-snapshots.qt.io/qt6-dev/linux.html
-        #
-        # In short, OpenGL libraries and headers are required. Without
-        # installing this package, cmake won't find out the correct lib path.
-        if [ "${{ matrix.os }}" == "ubuntu-22.04" ] ; then \
-          sudo apt-get -qy install \
-              qt6-3d-dev xvfb \
-              libgl1-mesa-dev
-        fi
+        # # Install qt6 only with ubuntu-22.04
+        # # This page explains why we need libgl1-mesa-dev
+        # # https://doc-snapshots.qt.io/qt6-dev/linux.html
+        # #
+        # # In short, OpenGL libraries and headers are required. Without
+        # # installing this package, cmake won't find out the correct lib path.
+
+        # if [ "${{ matrix.os }}" == "ubuntu-22.04" ] ; then \
+        #   sudo apt-get -qy install \
+        #       qt6-3d-dev xvfb \
+        #       libgl1-mesa-dev
+        # fi
+
+        # # This has been replaced by the 'Install Qt' section below to manage qt6 versioning independently from the OS.
```

Comment:
Added explanation for disabling here.

Reply:
Explanation should go before the commented-out code.